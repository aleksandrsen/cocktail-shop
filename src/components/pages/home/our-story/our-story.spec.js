import React from "react";
import { shallow, render } from "enzyme";
import OurStory from "./our-story";

describe("OurStory component", () => {
  it("should render OurStory component", () => {
    const component = shallow(<OurStory />);
    expect(component).toMatchSnapshot();
  });
});
