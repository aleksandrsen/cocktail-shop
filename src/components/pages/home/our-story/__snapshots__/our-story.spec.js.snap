// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OurStory component should render OurStory component 1`] = `
<div
  className="default-section ourStory"
>
  <div
    className="container"
  >
    <div
      className="row space-between align-items-center"
    >
      <div
        className="col col-5 col-sm-12"
      >
        <div
          className="section-title"
        >
          Our Story
        </div>
        <h2
          className="section-sub-title"
        >
          Lorem ipsum dolor sit amet, consectetur adipisicing elit.
        </h2>
        <p
          className="default-text"
        >
          Lorem ipsum dolor sit amet, consectetur adipisicing elit. A beatae dolorum eos illo quia quibusdam reprehenderit similique voluptate! Atque aut modi odio omnis? Animi assumenda doloremque obcaecati sapiente totam, ut. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Adipisci consectetur doloremque ea eligendi eveniet molestias mollitia reprehenderit! Accusamus atque, esse hic laborum maxime nulla odio possimus quas repudiandae voluptas voluptate.
        </p>
        <Link
          className="default-button"
          to="/about"
        >
          Read more
        </Link>
      </div>
      <div
        className="col col-5"
      >
        <img
          alt="Our story img"
          src="our-story.png"
        />
      </div>
    </div>
  </div>
</div>
`;
